website/components/learn-nomad/index.jsx

```diff
@@ -0,0 +1,58 @@
+import Button from '@hashicorp/react-button'
+
+export default function LearnNomad({ items }) {
+  return (
+    <div className="g-learn-nomad">
+      <div className="g-grid-container">
+        <div className="column-container">
+          <div className="column info">
+            <div className="column-content">
+              <h2 className="g-type-display-2">
+                Learn the latest Nomad skills
+              </h2>
+              <Button
+                className="desktop-button"
+                title="Explore HashiCorp Learn"
+                url="https://learn.hashicorp.com/nomad"
+                linkType="outbound"
+                theme={{ variant: 'primary', brand: 'nomad' }}
+              />
+            </div>
+          </div>
+          {items.map(item => {
+            return (
+              <a
+                key={item.title}
+                href={item.link}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <div className="column course">
```

Comment:
Same with this `.course` class

Reply:
Created a ticket here, will handle post launch, have a general polish ticket for this.